Sources/GDSCommon/UI/UIKit/SharedViews/Patterns/Results/ResultsViewModel.swift

```diff
@@ -0,0 +1,12 @@
+import UIKit
+
+/// Protocol for the view model required to initilise a ``ResultsViewController``
+public protocol ResultsViewModel {
+    var image: String { get }
+    var title: GDSLocalisedString { get }
+    var body: GDSLocalisedString? { get }
```

Comment:
I realise this PR has now already been merged...

Is there a reason for the `body` property being optional @Magdaline-A ?